deltachat-ffi/deltachat.h

```diff
@@ -1273,7 +1273,8 @@ void            dc_marknoticed_chat          (dc_context_t* context, uint32_t ch
  *
  * @memberof dc_context_t
  * @param context The context object as returned from dc_context_new().
- * @param chat_id The chat ID to get all messages with media from.
+ * @param chat_id >0: get messages with media from this chat ID.
+ *    0: get messages with media from any chat.
```

Comment:
Shall we add `of the currently used account` to  make clear the search is global within a given account?